### PR TITLE
Blog

### PR DESCRIPTION
**Pull request by mlrechtien**

### CODE REVIEW

File: src/pages/2022-09-02-davis-brin-dane-evans/index.md [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+
+So, let's say as an example that Tulsa just received the opening kickoff and took a knee in the end zone. It would be 1st and 10 from their own 25. Based on the model linked above, the expected points for that drive at the first play would be **0.922**. 
+
+Knowing this starting expected value allows us to determine how many added points the offense is able to generate on that drive. A touchdown would see the team add **6.078** points (TDs are assigned seven points for the sake of simplicity). A field goal would be **2.078**, and any sort of turnover during the drive – interception, fumble, punt – would yield **-0.922** points added because you got no points on the drive.
+
+I’ll break that into a table which might be a little easier to follow. Note that the expected points are all the same since it’s just based on the starting field position of the drive.
+
+<center>
+
+| | Expected Points | Acutal Points | EPA |
```
Comment (ryan-token) [inline src/pages/2022-09-02-davis-brin-dane-evans/index.md]: "Acutal" typo

File: src/pages/2022-09-02-davis-brin-dane-evans/index.md [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+
+Turning our attention to the final seven games, we can see that the lows aren't quite as low and there is more consistency betwen games.
+
+![Davis Second Half EPA](/blog_images/davis-brin/brin_second_season_epa.png)
+
+His performances with EPAs under zero -Navy, Cincy and Tulane- combined were barely worse than just the Houston game in terms of total EPA. So his bad games were not as bad–-and we won one of those games with a chance to play spoiler at Cincinatti. Additionally, the trend on the back half shows a much steeper improvement from game seven to thirteen. This gives me confidence that Davis Brin had a very normal 'first-year starter' season and makes me excited to see his progression in year two.
+
+I mean, the last time Tulsa had a first-year starter throw 16+ interceptions, with just over 3,000 yards of offense - it worked out pretty well. 
+
+## Dane Evan's First Season
```
Comment (ryan-token) [inline src/pages/2022-09-02-davis-brin-dane-evans/index.md]: This should be "Dane Evans'" or "Dane Evans's"

File: src/pages/2022-09-02-davis-brin-dane-evans/index.md [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+
+| **Completions** | **Attempts** | **Percentage** | **Yards** | **Yards / Attempt** | **Touchdowns** | **Interceptions** |
+| :---: | :---: | :---: | :---: | :---: | :---: | :---: |
+| 305 | 485 | 62.9 | 4,332 | 8.9 | 25 | 8 |
+
+</center>
+
+Most of those stats led the America in 2015 and Tulsa went from scoring 24.7 points a game to 37.2. And that was an exciting year for Tulsa - capped off by a bowl game against Virginia Tech that featured four touchdown within the first three and a half minutes. The momentum from that season carried forward to 2016 as Tulsa set an FBS record with two thousand yard rushers, two thousand yard receivers and a three thousand yard passer. 
+
+When Monty finds ***his*** QB he can win. And comparing Brin's latest season to Evan's first, it looks like he found his man.
```
Comment (ryan-token) [inline src/pages/2022-09-02-davis-brin-dane-evans/index.md]: Same comment as above about Evan's

File: src/pages/2022-09-02-davis-brin-dane-evans/index.md [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+|Zach Smith - 2019 | 0.247 | 106.0 |
+|Zach Smith - 2020 | 0.193 | 53.6* |
+|**Dane Evans - 2014** | 0.160 | 73.9 |
+|Skipper / Boomer - 2018 | 0.105 | 36.2 |
+|President / Skipper - 2017 | 0.072 | 29.2 |
+<br>
+**COVID shortened season*
+</center>
+<br>
+Evan's 0.160 EPA in 2014 was well below the 0.196 average for all of QBs in college football in 2014. Whereas Brin's 0.267 in 2021 was just above the average of 0.264 for the season. And against a more competitive schedule to boot.
```
Comment (ryan-token) [inline src/pages/2022-09-02-davis-brin-dane-evans/index.md]: Same comment as above about Evan's

File: src/pages/2022-09-02-davis-brin-dane-evans/index.md [hunk 5]
```diff
@@ -0,0 +1,215 @@
...
+
+| **Completions** | **Attempts** | **Percentage** | **Yards** | **Yards / Attempt** | **Touchdowns** | **Interceptions** |
+| :---: | :---: | :---: | :---: | :---: | :---: | :---: |
+| 256 | 462 | 55.4 | 3,102 | 6.7 | 23 | 17 |
+
+</center>
+
+<br />
+
+The only stat Evans lead Brin in was touchdowns with 23 to 18. Now, there is the caveat that 2014 was the final year with Bill Blankenship as head coach and he ran a different style compared to what Montgomery would bring the next season so its not exactly an apples to apples comparison. But that's why I spent so much time explaining EPA--so that I can make some sort of comparison.
```
Comment (ryan-token) [inline src/pages/2022-09-02-davis-brin-dane-evans/index.md]: The paragraph says Evans had 23 TDs vs 18 INTs, but the table says 23 TDs vs 17 INTs. Which is it?

Comment (mlrechtien) [reply src/pages/2022-09-02-davis-brin-dane-evans/index.md]: The 18 in that sentence is referring to Brin's TDs not Dane's INTs

Comment (ryan-token) [reply src/pages/2022-09-02-davis-brin-dane-evans/index.md]: Ah yeah whoops, that makes sense. Also I think "lead" should be "led" in this same paragraph.

Comment (mlrechtien) [reply src/pages/2022-09-02-davis-brin-dane-evans/index.md]: Probably, I kept going back and forth in my head lol

**Pull request merged**